Statistics, Exercise 1

Jim O'Regan

Question 1:

In [2]:
# Mean
63 / 300

0.21

Standard error:

$\sqrt{\frac{0.21(1-0.21)}{300}}$

In [6]:
import math
math.sqrt((0.21 * (1 - 0.21)) / 300)

0.023515952032609692

95% confidence limits $=0.21 \pm (1.96 \times 0.02)$

In [8]:
1.96 * 0.02

0.0392

$= 0.21 \pm 0.04$

$=$ **0.17** to **0.25**

Question 2:

In [9]:
533 / 4063

0.13118385429485602

$0.1312 \pm 1.96 \times \sqrt{\frac{0.1312(1-0.312)}{4063}}$

In [10]:
math.sqrt((0.1312 * (1 - 0.1312)) / 4063)

0.005296676057508483

In [11]:
1.96 * 0.0053

0.010388

$0.1312 \pm 0.0103$

**0.1209** to **0.1415**

Exercise 3

In [12]:
x1 = [9.5, 10.5, 9.0, 9.75, 10.0, 13, 10, 13.5, 10, 9.5, 10, 9.75]
x2 = [12.5, 9.5, 13.5, 13.75, 12.0, 13.75, 12.5, 9.5, 12.0, 13.5, 12.0, 12.0]

$H_0: \mu_1 = \mu_2$

$H_1: \mu_1 \ne \mu_2$

In [14]:
import numpy as np
mean1 = np.mean(x1)
mean2 = np.mean(x2)
mean1, mean2

(10.375, 12.208333333333334)

$\bar{X_1} = 10.375$

$\bar{X_2} = 12.2083$

$X$

$X~~~Y$

$X \rightarrow Y$

Form can be explained as a straight line; if the data is not a straight line, a transformation can be used.

$X \leftrightarrow Y$

The other model is the correlation model: both $X$ and $Y$ are random.

$X_1, X_2, X_3, \dots X_k, Y$

$Y = \alpha + \beta X + \epsilon$

($Y = \beta_0 + \beta_1 X + \epsilon$)

$\rho$

